js/app.js

```diff
@@ -1,63 +1,70 @@
 'use strict';
 
-var operatingHours = [
+var hours = [
   '6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm',
   '6pm', '7pm'];
 
 function Locations(name, minCustomer, maxCustomer, avgCookie) {
   this.name = name;
-  this.minCustomer = minCustomer;
-  this.maxCustomer = maxCustomer;
-  this.avgCookie = avgCookie;
+  this.minCustomerPerHour = minCustomer;
+  this.maxCustomerPerHour = maxCustomer;
+  this.avgCookiesPerCustomer = avgCookie;
   this.customersPerHour = [];
   this.cookiesPerHour = [];
+  this.totalCookies = 0;
 }
 
 Locations.prototype.calcCustomersPerHour = function(){
   for(var i = 0; i < hours.length; i++){
     var customersThisHour = getRandomNumber(this.minCustomerPerHour, this.maxCustomerPerHour);
-    this.customersEachHour.push(customersThisHour);
+    this.customersPerHour.push(customersThisHour);
   }
 };
+// helper function
+function getRandomNumber(min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
 
 Locations.prototype.calcCookiesSoldEachHour = function(){
   // multiply the customers by the average cookies each customer buys
-  for(var i = 0; i < this.customersEachHour.length; i++){
-    var wholeCookiesSoldEachHour = Math.ceil(this.customersEachHour[i] * this.avgCookiesPerCustomer);
+  for(var i = 0; i < this.customersPerHour.length; i++){
+    var wholeCookiesSoldEachHour = Math.ceil(this.customersPerHour[i] * this.avgCookiesPerCustomer);
     this.cookiesPerHour.push(wholeCookiesSoldEachHour);
     this.totalCookies += wholeCookiesSoldEachHour;
   }
 };
 
+Locations.prototype.render = function(){
+  this.calcCustomersPerHour();
+  this.calcCookiesSoldEachHour();
+  var tableElement = document.getElementById('table');
+  var rowElement = document.createElement('tr');
+  var rowHeader = document.createElement('th');
+
+
+  rowHeader.textContent = this.name;
+  rowElement.appendChild(rowHeader);
+  for(var i = 0; i < hours.length; i++){
+    var rowData = document.createElement('td');
+    rowData.textContent = this.cookiesPerHour[i];
+    rowElement.appendChild(rowData);
+  }
+  rowData = document.createElement('td');
+  rowData.textContent = this.totalCookies;
+  rowElement.appendChild(rowData);
+
+  tableElement.appendChild(rowElement);
+};
 
```

Comment:
The hours need to populate across the top of table with it ending with Daily Location Total